server/src/test/java/org/elasticsearch/cluster/node/DiscoveryNodesTests.java

```diff
@@ -420,8 +420,6 @@ public void testMaxMinNodeVersion() {
                 Version.fromString("1.1.0")
             )
         );
-        discoBuilder.localNodeId("name_1");
-        discoBuilder.masterNodeId("name_2");
         DiscoveryNodes build = discoBuilder.build();
```

Comment:
`name` should have been `node` here (not that it really matters)

```suggestion
        discoBuilder.localNodeId("node_" + between(1, 3));
        if (randomBoolean()) {
            discoBuilder.masterNodeId("node_" + between(1, 3));
        }
        DiscoveryNodes build = discoBuilder.build();
```